src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1787,7 +1743,10 @@ public FeedbackQuestionAttributes createFeedbackQuestionForTemplate(
      * Preconditions: <br>
      * * All parameters are non-null.
      */
-    public FeedbackQuestionAttributes copyFeedbackQuestion(String feedbackQuestionId, String feedbackSessionName, String courseId, String instructorEmail)
+    public FeedbackQuestionAttributes copyFeedbackQuestion(String feedbackQuestionId, 
+                                                           String feedbackSessionName, 
+                                                           String courseId, 
+                                                           String instructorEmail)
```

Comment:
When you have many parameters, consider having multiple parameters in the same line. In this case, we can have 2 per line, saving two lines.